assets/xml/textures/icon_item_gameover_static.xml

```diff
@@ -1,9 +1,9 @@
 <Root>
     <File Name="icon_item_gameover_static">
-        <Texture Name="Game_Over" Format="ia8" Width="64" Height="96"/>
-		<Texture Name="Game_Over_Mask" Format="ia8" Width="64" Height="32"/>
-		<Texture Name="Continue_Playing" Format="ia8" Width="152" Height="16"/>
-		<Texture Name="Spiel_Fortsetzen" Format="ia8" Width="152" Height="16"/>
-		<Texture Name="Continuer_La_Partie" Format="ia8" Width="152" Height="16"/>
+        <Texture Name="gGameOverTex" OutName="game_over" Format="ia8" Width="64" Height="96"/>
+        <Texture Name="gGameOverMaskTex" OutName="game_over_mask" Format="ia8" Width="64" Height="32"/>
+        <Texture Name="gContinuePlayingENGTex" OutName="continue_playing_eng" Format="ia8" Width="152" Height="16"/>
+        <Texture Name="gContinuePlayingGERTex" OutName="continue_playing_ger" Format="ia8" Width="152" Height="16"/>
+        <Texture Name="gContinuePlayingFRTex" OutName="continue_playing_fr" Format="ia8" Width="152" Height="16"/>
```

Comment:
I think we should probably have the same amount of letters for all languages, to remain consistent (eg. EN, DE and FR with 2 letters, or ENG, GER and FRA with 3). It's only one set of textures here so it's not too important, but we'll have to use a consistent pattern for everything so it's a discussion to have at least.